boot/zephyr/main.c

```diff
@@ -535,11 +561,16 @@ void main(void)
         /* at least one check if time was expired */
         timeout_in_ms = 1;
     }
-   boot_serial_check_start(&boot_funcs,timeout_in_ms);
+    boot_serial_check_start(&boot_funcs,timeout_in_ms);
```

Comment:
This is a legitimate fix, but probably should be in a separate commit.

Reply:
Moved to separate commit